scripts/data/delete-repeat-iterations.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# USAGE: ./delete-repeat-iterations.sh akvoflow-uat1 [surveyInstanceId] [group name] [1,2,4] [d | u]
```

Comment:
So in order to delete and update, you need to run the script twice? you cannot have both d and u?

Reply:
yes. thats the idea. I figured that if anything happens and there is a crash midway, then it will cause inconsistencies and delete unwanted data. e.g. if we delete iteration 3 and as we are updating the remaining ones i.e. 4 and 5 then something goes wrong, if we run the script again, we will then delete the updated ones from 4 and 5.